app/pages/api/auth/[...nextauth].ts

```diff
@@ -1,5 +1,5 @@
 import NextAuth, { NextAuthOptions } from "next-auth"
-import EmailProvider from "next-auth/providers/email"
+// import EmailProvider from "next-auth/providers/email"
```

Comment:
Could this be commented out by default? I often use the dev app to check something, without an adapter